### ISSUE TITLE
Supporting multiple languages/dictionaries

### ISSUE THREAD

**Issue by ChrisB9** [avojak/warble#14]
Thank you for this great application.
While diving into the source code I realized it would be pretty nice to support not only english words but a variety of other languages.

For example, instead of having the file called dictionary.txt, it could be called dictionary.<iso-code>.txt (same with words.txt), and then through a language selection menu it would be possible to switch languages.

I did do some research for my own language's word list and found this: https://github.com/enz/german-wordlist or https://gist.github.com/MarvinJWendt/2f4f4154b8ae218600eb091a5706b5f4. I could create two lists out of it, a dictionary with only 5 letter words and then a word list with roughly the same amount of words provided in the english set.

We then could to the exact same thing with french, spanish, or any other language that can be provided through a PR.
(ex. https://github.com/words/an-array-of-spanish-words, https://github.com/words/an-array-of-french-words)

**Comment by avojak** [comment]
I thought about this for a little bit initially during development and my biggest concern was handling the UI-keyboard layout. I don't have any experience with non US English keyboards so I'd have to figure that out. 

Thanks for pointing out those word lists, that's a good resource!

**Comment by ChrisB9** [comment]
That is a good point, I didn't even think about the keyboard layout.

The way I could imagine that, would be, to have a third file for that language, that contains the layout (and in the future maybe even translations), eg german could look like this:

```json
{
    "row1": {"q", "w", "e", "r", "t", "z", "u", "i", "o", "p", "ü"},
    ...
}
```

**Comment by avojak** [comment]
Good suggestion, definitely worth looking into further. 

I don't know any German, but are there any nuances with the German language that would complicate the game mechanics? I don't have any examples in mind, just a general question.

**Comment by ChrisB9** [comment]
I don't think so, the game mechanics should not get more complicated. As long as utf-8 characters can be displayed, the game should work the same
